packages/apex-lsp-vscode-extension/package.json

```diff
@@ -261,29 +261,25 @@
     },
     "commands": [
       {
-        "command": "apex.restart.server",
+        "command": "apex-ls-ts.restart.server",
         "title": "%commands.apex-ls-ts.restart.server.title%"
       },
       {
-        "command": "apex.setLogLevel.error",
+        "command": "apex-ls-ts.setLogLevel.error",
         "title": "%commands.apex-ls-ts.setLogLevel.error.title%"
       },
       {
-        "command": "apex.setLogLevel.warning",
+        "command": "apex-ls-ts.setLogLevel.warning",
         "title": "%commands.apex-ls-ts.setLogLevel.warning.title%"
       },
       {
-        "command": "apex.setLogLevel.info",
+        "command": "apex-ls-ts.setLogLevel.info",
         "title": "%commands.apex-ls-ts.setLogLevel.info.title%"
       },
       {
-        "command": "apex.setLogLevel.debug",
+        "command": "apex-ls-ts.setLogLevel.debug",
```

Comment:
No more overlapping 'apex' keywords for commands

Reply:
Yes, there was some same name config settings with 'apex' so this makes them unique